packages/form-generator/src/local_codegen_graphql_form_generator.test.ts

```diff
@@ -81,6 +81,12 @@ void describe('LocalCodegenGraphqlFormGenerator', () => {
           )
         );
       });
+
+      assert(
```

Comment:
see https://github.com/aws-amplify/amplify-backend/blob/065614f1d7c7d86a357add976b9749252f9e6517/packages/form-generator/src/local_codegen_graphql_form_generator.test.ts#L106-L107 for an example of making this assertion without modifying the public api

Reply:
Ah gotcha.